packages/former/src/components/FormKitEdit.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div v-if="mode === 'edit'" class="group flex flex-col handle">
+    <div class="flex gap-2 group-hover:border-blue-600 border border-transparent p-2 rounded-lg bg-white">
+      <div>
+        <FormKit v-bind="$attrs" />
+      </div>
+
+      <div class="ml-auto gap-4 hidden group-hover:flex">
+        <FormKit type="button" label="Edit" :onClick="edit" />
+        <FormKit type="button" label="Delete" :onClick="onDelete" />
+      </div>
+    </div>
+
+    <div class="mx-auto mt-2 hidden group-hover:flex">
+      <FormKit type="button" label="Add component" :onClick="addAfter" />
```

Comment:
@anbraten And all these buttons are not visible. Is this related to the not working tailwind config? Seems to be the case for me...

Reply:
It's part of the tailwind issue.